src/utils/validatePassword.js

```diff
@@ -0,0 +1,11 @@
+const validatePassword = (password) => {
+  if (!password) {
+    return 'password is required';
+  }
+
+  if (password.length < 6) {
+    return 'at least 6 characters';
+  }
+};
```

Comment:
Consider returning 'null' or 'undefined' explicitly when the password is valid. This will make it clear that the function returns a string only when the password is missing or does not meet the length requirement.